### ISSUE TITLE
build-containers

### ISSUE THREAD

**Issue by jinseo-jang** [brentley/appmeshworkshop#104]
in build-containers scripts, docker build command should be adjusted to find proper Dockerfiles.
`docker build -t crystal-service ../ecsdemo-crystal` or `docker build -t crystal-service ~/environment/ecsdemo-crystal`

**Comment by jeeslo** [comment]
Is this an actual issue or an improvement? If I am not mistaken, following the workshop instructions you are intended to run the generated script from the ~/environment path, hence the docker build commands work as expected. In any case, I agree that its a quick change that could avoid the script failing out should someone decides to run it from somewhere else.

**Comment by jinseo-jang** [comment]
yes. it can be an improvement in case learners are just lost where to execute that script.
